### PR TITLE
TRIVIAL: finalize gooddata-dbt

### PR DESCRIPTION
**Pull request by jaceksan**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-dbt/gooddata_dbt/dbt/cloud.py [hunk 1]
```diff
@@ -17,6 +20,52 @@
...
+              ) {
+                runId # Get historical results for a particular model
+                runGeneratedAt
+                executionTime # View build time across runs
+                status
+                #tests {
+                #  name
+                #  status
+                #  executeCompletedAt
+                #} # View test results across runs
```
Comment (hkad98) [inline gooddata-dbt/gooddata_dbt/dbt/cloud.py:63]: Do we want to include these comment attributes?

**Review by hkad98 — APPROVED** [review]

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-dbt/gooddata_dbt/dbt/cloud.py (hunk 1 above)
Comment (jaceksan) [reply gooddata-dbt/gooddata_dbt/dbt/cloud.py]: Removed. Can be added any time later.

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR346

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-fossa-scanning-zuul/159/ : FAILURE in 1m 12s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-sonarqube-scan-zuul-gate/309/ : SUCCESS in 55s
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-merge-compliance-check/312/ : SUCCESS in 10s

**Pull request merged**